ace_studio_streamlit/main.py

```
@@ -0,0 +1,216 @@
+    st.divider()
+
+    # Quick project count
+    try:
+        from utils import ProjectManager
+        from config import PROJECTS_DIR
+
+        pm = ProjectManager(PROJECTS_DIR)
+        projects = pm.list_projects()
+        st.metric("💾 Projects", len(projects))
+    except Exception:
+        pass
```

Ruff (0.15.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)